devops/jobs/CreateSandboxCI.groovy

```diff
@@ -52,17 +52,70 @@ class CreateSandboxCI {
                 booleanParam('VERBOSE',true)
                 // Once we resolve the mismatched Elastic Search versions, enable discovery
                 //booleanParam('course_discovery',true)
-                if (type == 'Daily') {
-                  predefinedProp('server_type', 'full_edx_installation_from_scratch')
-                  predefinedProp('dns_name','int-nightly')
-                  predefinedProp('name_tag','continuous-integration-nightly')
-                } else {
+                if (type == 'Hourly') {
                   predefinedProp('dns_name','int')
                   booleanParam('reconfigure',true)
                   booleanParam('recreate',false)
                   predefinedProp('name_tag','continuous-integration')
                   booleanParam('ecommerce_worker',true)
                   booleanParam('credentials',true)
+                } else if (type == 'Daily') {
+                  predefinedProp('server_type', 'full_edx_installation_from_scratch')
+                  predefinedProp('dns_name','int-nightly')
+                  predefinedProp('name_tag','continuous-integration-nightly')
+
+                // Masters integration environment weekly CI.
+                // Monitored by Registrar supporting team:
+                // https://github.com/edx/registrar/blob/master/openedx.yaml
+                } else if (type == 'MastersWeekly') {
+                  predefinedProp('dns_name','masters-weekly')
+                  predefinedProp('name_tag','masters-weekly')
+
+                  // Define to 30 because:
+                  // * This will be re-built every week anyway
+                  // * We want real Masters integration boxes to be able to use
+                  //   these parameters as defaults.
+                  predefinedProp('sandbox_life','30')
```

Comment:
Why not define this to 7 then, since it will get wiped in 7 days without intervention?

Reply:
Do you want the `masters-weekly` box or the intergration environments to be wiped in 7 days without intervention?

Reply:
I think that's fine for now, because if the next rebuild succeeds, we still have the environment.

Reply:
We synced offline. And I am fine with having this parameter to be 30